### PR TITLE
313_shame-overrides

### PR DESCRIPTION
**Pull request by jamesjacobs**

### CODE REVIEW

File: stylesheets/_component.forms.scss [hunk 1]
```diff
@@ -6,6 +6,32 @@ fieldset {
 
 .form {
     background-color: color(white);
+
+    // Added by @stanton, not used in CXM - safe to remove?
+    .slide-inner & {
+        padding-top: 17px;
```
Comment (jamesjacobs) [inline stylesheets/_component.forms.scss]: Is this still required @Stanton?

Comment (Stanton) [inline stylesheets/_component.forms.scss]: I'm almost certain `.slide-inner` is a class from the old decks component, I can't find it in use in CMS or Pulsar so should be ok to remove

**Comment by jamesjacobs** [comment]
Fixes #313

**Comment by jamesjacobs** [comment]
Closes #313

**Pull request merged**